src/current/v23.1/common-issues-to-monitor.md

```diff
@@ -128,19 +128,20 @@ CockroachDB is [resilient]({% link {{ page.version.version }}/demo-fault-toleran
 
 Provision enough memory and allocate an appropriate portion for data caching:
 
-| Category | Recommendations                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
-|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Memory   | <ul><li>Provision at least {% include {{ page.version.version }}/prod-deployment/provision-memory.md %}.</li><li>{% include {{ page.version.version }}/prod-deployment/prod-guidance-cache-max-sql-memory.md %} For more details, see the [Recommended Production Settings]({% link {{ page.version.version }}/recommended-production-settings.md %}#cache-and-sql-memory-size).</li><li>{% include {{ page.version.version }}/prod-deployment/prod-guidance-disable-swap.md %}</li><li>See additional memory recommendations in the [Recommended Production Settings]({% link {{ page.version.version }}/recommended-production-settings.md %}#memory).</li> |
+- Provision at least {% include {{ page.version.version }}/prod-deployment/provision-memory.md %}.
```

Comment:
Move data out of single-row table

Reply:
This part seems off.
![Screenshot 2024-04-03 at 2 04 25 PM](https://github.com/cockroachdb/docs/assets/5135385/c043747a-cc52-4769-a002-91181c4dbad9)

Reply:
Thanks, this was complicated to get the same include to work in both the Recommended Production Settings and Common Issues to Monitor. Should be good now.